src/test/java/org/mitre/synthea/engine/ExpressedSymptomTest.java

```diff
@@ -0,0 +1,120 @@
+package org.mitre.synthea.engine;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+import org.mitre.synthea.engine.ExpressedSymptom.SymptomInfo;
+import org.mitre.synthea.engine.ExpressedSymptom.SymptomSource;
+
+public class ExpressedSymptomTest {
+
+  @Test
+  public void testSymptomInfo() {
+    String name = "pain";
+    ExpressedSymptom symptom = new ExpressedSymptom(name);
+    String cause = "test";
+    Integer value = 100;
+    Long time = 0L;
+
+    SymptomInfo info = symptom.new SymptomInfo(cause, value, time);
```

Comment:
I don't ever remember coming across this syntax before. I had to look it up. I guess it is because we always use `static` inner classes elsewhere in the project.